spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -310,6 +320,52 @@ def _make_figure(metric, fit_results):
     return fname_img
 
 
+def get_data_for_normalization(norm_args):
+    """
+    Get coefficients of multilinear regression for CSA normalization and subject data.
+    Normalization models are under data/csa_normalization_models/
+    Models are generated with https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/blob/master/pipeline_ukbiobank/cli/compute_stats.py
+    # TODO update link with release tag.
+
+    :param norm_args: arguments from -normalize.
+
+    :return data_predictors and data_subject:
+
+    """
+    PREDICTORS_DICT = {'brain-volume': 'brain volume', 'thalamus-volume': 'thalamus volume'}
+    # Select model
+    model_1 = ['sex', 'brain-volume']
+    model_2 = ['thalamus-volume', 'sex', 'brain-volume']
+    if all(item in norm_args for item in model_1):
+        model = 'coeff_brain_sex'
+    elif all(item in norm_args for item in model_2):
+        model = 'coeff_brain_thalamus_sex'
+    else:
+        raise ValueError('Invalid choice of predictors in -normalize. Please specify sex and brain-volume or sex, brain-volume and thalamus-volume')
+    path_model = os.path.join(__data_dir__, 'csa_normalization_models', model + '.csv')
```

Comment:
> I would like to put the statistical model under `data/csa_normalization_models`, this means creating a repo for the models, adding the option in `sct_download_data` and add the download in `install_sct`. However, it is only 2 .csv files. @kousu @joshuacwnewton would that be ok or do you have any other ideas to save the models?

Another approach mentioned by @kousu in the August 10th SCT meeting would be to keep the smaller model files within the SCT repo itself, since they are reasonable (size-wise) to have in the main repo.

I think either approach would work! I'd love to hear @kousu's input, too.

Reply:
Personally, I do like that having a separate repo gives us a link outside of the SCT repo as well as a dedicated landing page with a README.md, which is useful for papers, etc. (Hardcoded locations inside the SCT repo have [caused issues](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3461#discussion_r672328846) for us in the past.)

Reply:
I followed @kousu's suggestion and the created a data folder under `spinlcordtoolbox/data/`